pennylane_lightning/core/_serialize.py

```diff
@@ -59,7 +59,12 @@ class QuantumScriptSerializer:
 
     # pylint: disable=import-outside-toplevel, too-many-instance-attributes, c-extension-no-member, too-many-branches, too-many-statements
     def __init__(
-        self, device_name, use_csingle: bool = False, use_mpi: bool = False, split_obs: bool = False
+        self,
+        device_name,
+        use_csingle: bool = False,
+        use_mpi: bool = False,
+        split_obs: bool = False,
+        tensor_backend: str = "mps",
```

Comment:
Could you update this to  `backend`? We should be consistent with the UI of `"default.qubit"`.

https://github.com/PennyLaneAI/pennylane/blob/69bd9ff3e17f026c4dc9da8737d37ad09fc11e3f/pennylane/devices/default_tensor.py#L374C1-L375C1

Reply:
Done!